torch/_decomp/__init__.py

```diff
@@ -176,116 +176,152 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
     aten = torch.ops.aten
     return get_decompositions(
         [
-            aten.linspace,
-            aten.logaddexp,
             aten._adaptive_avg_pool2d_backward,
-            aten.addcmul,
-            aten.addcmul_,
+            aten.addcdiv,
             aten.addcdiv,
             aten.addcdiv_,
+            aten.addcmul,
+            aten.addcmul_,
+            aten.addr,
             aten.avg_pool2d_backward,
+            aten.binary_cross_entropy,
+            aten.binary_cross_entropy_backward,
             aten.binary_cross_entropy_with_logits,
+            aten.bucketize,
+            aten.celu,
             aten.col2im,
             aten.cudnn_batch_norm,
             aten.cudnn_batch_norm_backward,
             aten.detach,
+            aten.diag_embed,
+            aten.diagonal,
             aten.dot,
             aten.elu,
             aten.elu_backward,
             aten._embedding_bag,
             aten.embedding_dense_backward,
             aten.expand_as,
             aten.eye,
-            aten.ones_like,
-            aten.zeros_like,
-            aten.zeros,
-            aten.ones,
             aten.fill,
+            aten.frac,
             aten._fused_moving_avg_obs_fq_helper,
             aten.gelu,
             aten.gelu_backward,
             aten.glu_backward,
             aten.grid_sampler_2d,
+            aten.hardshrink,
+            aten.hardshrink_backward,
             aten.hardsigmoid,
             aten.hardsigmoid_backward,
-            aten.upsample_bilinear2d,
             aten.hardswish,
             aten.hardswish_,
             aten.hardswish_backward,
             aten.hardtanh,
             aten.hardtanh_,
             aten.hardtanh_backward,
+            aten.heaviside,
+            aten.huber_loss,
+            aten.huber_loss_backward,
             aten.im2col,
-            aten.index_select,
             aten.index_add,
             aten.index_add_,
             aten.index_copy,
             aten.index_copy_,
             aten.index_fill,
             aten.index_fill_,
-            aten.isposinf,
+            aten.index_select,
+            aten.isneginf,
             aten.isneginf,
+            aten.isposinf,
+            aten.isposinf,
+            aten.isposinf,
```

Comment:
What's going on here, is the duplication intentional?

Reply:
thanks, good catch, not sure how that happened